### DISCUSSION TITLE
Welcome to podbox Discussions!

### DISCUSSION THREAD

**Discussion by redbeardymcgee** [redbeardymcgee/podbox#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EphemeralDev** [comment]
Thanks for answering my questions in that other issue, really appreciate the help.

I have another question regarding your setup of the container user. Is it required/necessary to setup a password for that user? If it doesn't have sudo access the user can do the same things with or without a password, right? I would just prefer to not have to keep track of another password if its not strictly necessary or doesn't improve security.

**Reply by redbeardymcgee** [reply]
I prerfer to disable login for the container user entirely. I only access the user by connecting over SSH to my regular user, then `sudo -u $ctuser bash -l` or `ssh -t $host sudo -u $ctuser bash -l`.

**Reply by redbeardymcgee** [reply]
`machinectl shell ct@ /bin/bash`

This is now how I switch to my container user, which plays nicer with `ssh` as well

`ssh -t $hostname machinectl shell $ctuser@ /bin/bash`
